lib/web_ui/lib/window.dart

```diff
@@ -121,9 +121,28 @@ abstract class AccessibilityFeatures {
   bool get onOffSwitchLabels;
 }
 
+/// Describes the contrast of a theme or color palette.
 enum Brightness {
```

Comment:
To facilitate this pull request being merged as smoothly as possible, could we remove the documentation added to this enum, so it's consistent with the rest of this file?

Reply:
@nate-thegrate I'm sorry, but im not able to get my environment setup to generate the test's.

I just have no idea what to do, and I'm keep getting this error:

Resolving dependencies in `D:\flutter-engine`...
Because _engine_workspace depends on yaml from path which doesn't exist (could not find package yaml at "..\..\..\third_party\dart\third_party\pkg\yaml"), version solving failed.
exit code 66

And my dart anal keeps crashing:
![image](https://github.com/user-attachments/assets/02f1e169-313e-4667-ae5a-f2d79636b94f)

Reply:
![image](https://github.com/user-attachments/assets/0aa8588e-c353-49d7-9299-8d88b3d5cc49)

Reply:
> ![image](https://github.com/user-attachments/assets/02f1e169-313e-4667-ae5a-f2d79636b94f)

Loving the subtle watermark lol

<br>

Hopefully this resource can help: https://github.com/flutter/engine/blob/main/docs/contributing/Setting-up-the-Engine-development-environment.md

I'd also be happy to push a commit to this PR if you'd like :)

Reply:
@nate-thegrate Yes, please i did everything but i'm totally lost on the engine part...

Reply:
it seems like it expects yaml to be installed globally ?